src/services/main-serviсe.ts

```diff
@@ -0,0 +1,53 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+import { ArtworkDetails, getArtworksItemsResponse } from '../types/types';
+
+export const getAllItemsAPI = createApi({
+  reducerPath: 'getAllItemsAPI',
+  baseQuery: fetchBaseQuery({
+    baseUrl: 'https://api.artic.edu/api/v1',
```

Comment:
const BASE_URL = 'https://api.artic.edu/api/v1';
```suggestion
    baseUrl: 'https://api.artic.edu/api/v1',
    baseUrl: BASE_URL
```